packages/app-page-builder/src/editor/plugins/elementSettings/components/SelectField.tsx

```diff
@@ -61,7 +61,6 @@ const SelectField = ({
     return (
         <React.Fragment>
             <select
-                placeholder={placeholder}
```

Comment:
Why did you remove `placeholder`?

Reply:
HTML’s `select` tag does not have `placeholder` property. We already use [this](https://github.com/webiny/webiny-js/blob/next/packages/app-page-builder/src/editor/plugins/elementSettings/components/SelectField.tsx#L75) approach to add a placeholder.